core/subsurface-qt/diveobjecthelper.cpp

```diff
@@ -286,7 +308,9 @@ DiveObjectHelper::DiveObjectHelper(const struct dive *d) :
 	getCylinder(formatGetCylinder(d)),
 	startPressure(getStartPressure(d)),
 	endPressure(getEndPressure(d)),
-	firstGas(getFirstGas(d))
+	firstGas(getFirstGas(d)),
+        salinity(formatSalinity(d)),
```

Comment:
this looks like there is a whitespace issue

Reply:
*doh*